wp-cache.php

```diff
@@ -3688,16 +3686,29 @@ function wpsc_enable_preload() {
 	wp_schedule_single_event( time() + 10, 'wp_cache_full_preload_hook' );
 }
 
+function wpsc_get_post_types() {
+
+	$preload_type_args = apply_filters( 'wpsc_preload_type_args', array(
```

Comment:
I'm thinking about better name for filter. Maybe `wpsc_preload_get_pt_args` or `wpsc_preload_get_post_types_args` (it seems too long for me, I prefer first variant). Only types isn't totally clear which type?